src/testRunner/unittests/config/tsconfigParsing.ts

```diff
@@ -412,5 +412,14 @@ namespace ts {
                 Diagnostics.Compiler_option_0_requires_a_value_of_type_1.code,
                 /*noLocation*/ true);
         });
+
+        it("parses wildcard directories even when parent directories have dots", () => {
+            const parsed = parseConfigFileTextToJson("/foo.bar/tsconfig.json", JSON.stringify({
+                include: ["src"]
+            }));
+
+            const parsedCommand = parseJsonConfigFileContent(parsed.config, sys, "/foo.bar");
+            assert.deepEqual(parsedCommand.wildcardDirectories, { "/foo.bar/src": WatchDirectoryFlags.Recursive });
+        });

```

Comment:
Would it be worth adding a test for `/repo/src/example.com` as well? (Dot in the include path, but not in the repo path.)

Reply:
A dot in the last component of the include path is still going to cause a problem—I’m not sure we’ve settled on what the expected behavior for that is.